service/storage_service.cc

```diff
@@ -1486,6 +1486,17 @@ sstring storage_service::get_application_state_value(inet_address endpoint, appl
     return v->value;
 }
 
+std::optional<db_clock::time_point> storage_service::get_streams_timestamp_for(inet_address endpoint) {
```

Comment:
This method does not access storage_service internals. I suggest to move to cdc directory.  storage_service is crowded.

Reply:
It calls `get_application_state_value`, which is private

Reply:
It should be a public method of gossiper though

Reply:
Fixd